index.html

```diff
@@ -210,18 +210,17 @@ <h3 id="matters-considered-by-the-placement-rules">Considerations for  simple pl
     Specifically, the center of the ruby string and of the base character 
     string are aligned in the inline direction for mono-ruby.</li>
 <li id="l20200529009">Processing is done in two levels.
-    In the first level, processing of layout only considers the relative position of the ruby annotation and its base text. 
-    In the second level, layout processing decides the position of the <a>ruby box</a> in the line. 
+    The first level only determines relative positions of ruby annotation and its base. 
+    Meanwhile, the second level determines inline positions of <a>ruby boxes</a>. 
     In other words, relative positions of the ruby annotation and its base text as decided in the first level 
     are not modified by the second level such as in light of any characters or line edge preceding and following the <a>ruby box</a>.
-    Two levels are processed as following, with using <dfn>ruby column</dfn> for paired ruby annotation and its base text, 
-    and untransparent <a>ruby box</a> for <a>ruby box</a> with their relative position:
+    Two levels are processed as following:
     <ol>
-    <li id="l20220318001">The first level takes a <a>ruby column</a> with required parameters for 
-        processing placement for inline direction inluding available space within the current line 
-        but without any preceding or following characters, 
-        and outputs one or more untransparent <a>ruby box</a>.</li>
-    <li id="l20220318002">The second level takes a size and preceding and following text of <a>ruby boxes</a>, 
+    <li id="l20220318001">The first level takes pairs of ruby annotation and its base text 
```

Comment:
Suggestion:
The first level takes pairs of ruby base and its annotation, including parameters needed for processing, such as available space in the inline direction of the current line. The first level outputs one or more [ruby boxes], that indicate the position of the ruby annotation with respect to its base.
The second level looks at the size of untransparent [ruby box] and its surrounding text, and outputs styled text.

Reply:
I'm not clear what 'untransparent' means (i haven't looked around the document for an explanation).

Reply:
Thank you for comment on this. For 'untransparent', there is still discussion on terminology and explanation even in Japanese, and we are seeking better (and clearer?) text for this part...

Reply:
text updated per discussion within JL-TF. 'untransparent' has gone.